examples/chip-tool/commands/pairing/PairingCommand.cpp

```diff
@@ -403,6 +403,29 @@ void PairingCommand::OnCommissioningComplete(NodeId nodeId, CHIP_ERROR err)
     SetCommandExitStatus(err);
 }
 
+void PairingCommand::OnReadCommissioningInfo(const Controller::ReadCommissioningInfo & info)
+{
+    ChipLogProgress(AppServer, "OnReadCommissioningInfo - vendorId=0x%04X productId=0x%04X", info.basic.vendorId,
+                    info.basic.productId);
+
+    if (info.isLIT)
```

Comment:
Can you open a follow up issue to Verify if we want to keep these behaviors behind the isLit flag?

Reply:
based upon discussion, for SIT, it is optional, but if exists, we need to tell application

Reply:
Now this is changed to `info.icdUserActiveModeTriggerHint.HasAny()`